{{cookiecutter.repo_name}}/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016, Qiita Spots
```

Comment:
We discussed this offline and we are gonna offer 3 license types by default: BSD 3-clause, MIT and Apache. Developers are always free of changing the license afterwards.